Sources/SwiftProtobuf/Google_Protobuf_Timestamp_Extensions.swift

```diff
@@ -191,14 +195,12 @@ private func parseTimestamp(s: String) throws -> (Int64, Int32) {
 }
 
 private func formatTimestamp(seconds: Int64, nanos: Int32) -> String? {
-    if ((seconds < 0 && nanos > 0)
-        || (seconds > 0 && nanos < 0)
-        || (seconds < -62135596800)
-        || (seconds == -62135596800 && nanos < 0)
-        || (seconds >= 253402300800)) {
-            return nil
+    guard isWithinValidTimestampRange(seconds: seconds, nanos: nanos) else {
+        return nil
     }
 
+    let (seconds, nanos) = normalizeForTimestamp(seconds: seconds, nanos: nanos)
```

Comment:
isWithinValidTimestampRange also calls normalizeForTimestamp, should we just normalize first, and then validate and make validate skip that step?